advertising101.md

```diff
@@ -0,0 +1,167 @@
+
+* I onboard my segmented CRM files, which include this customer’s information, into my DMP of choice, and into my DSP of choice. Based on the matches between my onboarder and my DMP and DSP respectively, I do not know if this particular customer is matched or not, but I have a high likelihood because I achieved a 90% match rate with my DSP and a 96% match rate with my DMP (I assessed the onboarder, DMP and DSP that I selected each based on match rates)
+* Let’s say that I ran analysis of my data and it showed that customers make their second purchases of diapers between days 18-23. I can remarket to this customer from days 12-23, setting frequency caps during that period so as not to inundate the customer and annoy him (and waste my money), but ensuring that I am talking to him at a time when he likely needs to purchase his next set of diapers
+* Then when this customer is online during this time period, my DSP will target him using the pseudonymized identifier that they have in their identity graph and decide whether or not to bid on him based on:
+  1. price
+  2. frequency caps (determined through the DSP’s identity graph across all known devices attached to each individual consumer)
+  3. brand safety
+* I will not learn whether this particular customer was marketed to, but I will be able to understand in aggregate the effectiveness of my marketing based on lift measurement results of the campaign (this understanding is vital to me as without it, it’s very hard for me to get my finance team to approve budgets)
+
+
+**_Scenario Four: Re-engagement_**
+
+*Marketer*: My customer has now not purchased with me in a year, which moves him into the lapsed customers segment. However, I would like to regain him as a customer. I have tried marketing directly to him via email and direct mail, but have not seen any resultant conversions. Because I know he is heavily digital in his day-to-day, I think that marketing to him on those channels will be more relevant to him
+* I repeat the steps from relationship marketing, but this time elect to trial different messages using both dynamic creative optimization and adaptive segments because I want to react in real time based on his behaviors, tailoring my campaign and message based on what he interacts with or not
+* I also set higher frequency caps and reach out to “him” on more digital channels than I have previously (via the segment – everyone in the segment is treated the same initiall,y and again, I don’t understand tindividuals tied to my CRM profiles). I can do this because my DSP has good understanding of the various devices attached to the individuals in my segment
+* Based on attribution run through the DSP using the household level identity graph, the DSP is able to run lift measurement and understand that the ads led to a conversion at the household level, not at the individual level (the individual who saw the ads is not the one who made the purchase, but the HH did make a purchase)
```

Comment:
What data signals do DSPs use to identify households? I'm guessing location/IP, but are there others?

Reply:
Will make sure to cover this in subsequent iterations which ladder down to technical mechanisms that implement business goals!